components/gitpod-protocol/go/reconnecting-ws.go

```diff
@@ -181,7 +175,13 @@ func (rc *ReconnectingWebsocket) Dial(ctx context.Context) error {
 }
 
 func (rc *ReconnectingWebsocket) connect(ctx context.Context) *WebsocketConnection {
-	delay := rc.minReconnectionDelay
+	exp := backoff.NewExponentialBackOff()
+	exp.InitialInterval = 2 * time.Second
+	exp.RandomizationFactor = 0.5
+	exp.MaxInterval = 30 * time.Second
+	exp.Multiplier = 1.5
+	exp.MaxElapsedTime = 0
+	exp.Reset()
```

Comment:
I do not quite understand why we need to call `Reset()` function. Could you please explain a bit more? Thank you.

Reply:
because docs says so: https://github.com/cenkalti/backoff/blob/6b0e4ad0cd65431b217393bff47d1dff727b264b/exponential.go#L108

otherwise startTime is not initialized